<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>menu.jpg</normaloff>menu.jpg</iconset>
  </property>
  <widget class="QLabel" name="label_excluir_cliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4-Excluir cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_excluir_user">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>menu-botoes.png</normaloff>menu-botoes.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_menu">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pesquisar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>menu-botoes.png</normaloff>menu-botoes.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_visualizar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>menu-botoes.png</normaloff>menu-botoes.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_pesquisar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3-Pesquisar cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cadastrar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1-Cadastrar cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cadastrar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>menu-botoes.png</normaloff>menu-botoes.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_exclui_user">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6-Excluir usu√°rio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_editar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2-Editar cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_relatorio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5-Visualizar clientes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_editar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>menu-botoes.png</normaloff>menu-botoes.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_excluir_cliente">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>menu-botoes.png</normaloff>menu-botoes.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
